src/ape/api/transactions.py

```diff
@@ -294,6 +294,25 @@ def validate_txn_hash(cls, value):
     def call_tree(self) -> Optional[Any]:
         return None
 
+    @cached_property
+    def debug_logs_typed(self) -> List[Tuple[Any]]:
+        """Return any debug log data outputted by the transaction"""
```

Comment:
nit: most of our docs end in periods i think

```suggestion
        """Return any debug log data outputted by the transaction."""
```